packages/shared_preferences/shared_preferences_tool/lib/src/ui/shared_preferences_body.dart

```diff
@@ -28,3 +39,48 @@ class SharedPreferencesBody extends StatelessWidget {
     );
   }
 }
+
+class _WebUnavailableMessage extends StatelessWidget {
+  const _WebUnavailableMessage();
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+
+    return Center(
+      child: Container(
+        width: 400.0,
+        padding: const EdgeInsets.all(largeSpacing),
+        child: Text.rich(
+          textAlign: TextAlign.center,
+          TextSpan(
+            children: <InlineSpan>[
+              TextSpan(
+                text: 'The shared preferences tool is not available for web '
+                    'platforms yet. Once this ',
+                style: theme.regularTextStyle,
+              ),
+              TextSpan(
```

Comment:
@kenzieschmoll I know that you have the [LinkTextSpan](https://github.com/flutter/devtools/blob/f5e920d73b1544b11e50fbc72d931695cc325120/packages/devtools_app_shared/lib/src/ui/common.dart#L903), but since it is not release on a stable version, I had to add it manually here.

Reply:
can you use `devtools_app_shared` and  `devtools_extensions` version `0.2.0-dev.0` so that you can use the latest from both packages?

Reply:
Should we use a `dev` dependency, though?

Reply:
These packages are all non-dev versions now, so I recommend bumping to the latest versions for this extension. All are compatible with the current Flutter stable.
- devtools_extensions: 10.0.1
- devtools_app_shared: 0.2.1